spec/models/user_spec.rb

```diff
@@ -1,7 +1,7 @@
 require 'rails_helper'
 
 RSpec.describe User, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+  "add some examples to (or delete) #{__FILE__}"
```

Comment:
One last change here: please delete line 4 and instead move lines 7-17 to be INSIDE this block